sentry-kotlin-multiplatform/src/androidMain/kotlin/io/sentry/kotlin/multiplatform/SentryInit.kt

```diff
@@ -0,0 +1,16 @@
+package io.sentry.kotlin.multiplatform
+
+import android.content.Context
+import io.sentry.android.core.SentryAndroid
+import io.sentry.kotlin.multiplatform.extensions.toAndroidSentryOptionsCallback
+
+actual fun initJvmTarget(context: Context?, configuration: OptionsConfiguration) {
```

Comment:
Should this method be renamed to initAndroidTarget?

Reply:
`Context` cannot be nullable.

Reply:
I don't think I can change a function name with expect/actual, expect defines the function that the submodules have to implement with actual and the names and signatures have to match.

both jvmMain and androidMain implement actual fun initJvmTarget, maybe I can rename it to something more generic?

Reply:
A non nullable Context is hard to implement since multiple platforms share initializer signatures and it's hard to force users only to use one specific initializer. For example it's possible on Android to use  `Sentry.init { ... }` which is incorrect